README.md

```diff
@@ -86,14 +86,26 @@ By selecting quick replies instead of typing manually, agents/users can respond
 
 -   **`/quick`**: Get started with Quick Reply
 -   **`/quick create`**: Create a new quick reply
+-   **`/quick create "<name>" <message>`**:  Create a quick reply directly from the message box with a name and message
 -   **`/quick list`**: List all your quick replies
 -   **`/quick config`**: Configure your language preferences and AI settings
 -   **`/quick ai`**: Use AI to generate replies
 -   **`/quick help`**: Get help with Quick Reply
 -   **`/qs <reply name>`**: Quickly search and send a reply by name
 
-### Using Placeholders:
 
+#### Creating Quick Replies from Message Box
+**`/quick create "<name>" <message>`**: "if your name has multiple words with space, use double quotes" and write everything else as a body!
+
+Example: `/quick create "schedule meeting" let's have a quick meet in an hour!`
+
```

Comment:
This is not what I meant.

Can you please avoid writing a lot in the README to explain really simple things?